### PR TITLE
README.md: update homepage link

### PR DESCRIPTION
**Pull request by nafg**

### CODE REVIEW

**Review by swoogles — APPROVED** [review]
I literally hit this bad link moments ago. 
Thanks!

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ You can use the [sbt-revolver] plugin to start the server and run it in watch mo
 
 ## Documentation
 
-Learn more on the [ZIO Http homepage](https://github.com/zio/zio-http)!
+Learn more on the [ZIO Http homepage](https://zio.dev/zio-http/)!
```
Comment (khajavi) [inline README.md:55]: Please don't touch the README.md directly. It is autogenerated.

Comment (khajavi) [reply README.md:55]: Instead, change `homepage` setting in the `build.sbt` file.

**Comment by nafg** [comment]
Maybe the readme should say that

File: README.md (hunk 1 above)
Comment (nafg) [reply README.md:55]: Maybe the readme should say that not as a comment but a visible text so I know not to edit it directly in the web UI

Comment (khajavi) [reply README.md:55]: @nafg Do you have any idea how to notify contributors not to touch the readme?

Comment (nafg) [reply README.md:55]: The first line of the rendered readme should say so. Basically what's now a comment should be made visible in the rendered view. Perhaps in a small italicized font.

Comment (khajavi) [reply README.md:55]: Thank you! Anyway, please let me know if you are willing to finish this work or not.

Comment (nafg) [reply README.md:55]: Something came up and I'm really busy for now, so if you could take it over it would be great.

Comment (khajavi) [reply README.md:55]: I will do that!

**Comment by nafg** [comment]
I see build.sbt is updated but the link in the README is still incorrect

**Pull request closed without merging**